web/modules/custom/foreningsmentor/templates/signup-email-body.html.twig

```diff
@@ -0,0 +1,26 @@
+<h1>Ny tilmelding</h1>
+<p>Der er kommet en ny tilmelding på {{ params.site_name }}.</p>
+<h2>Barnets information</h2>
+<h4>Barnets navn</h4>
```

Comment:
The next header level after 2 is 3, i.e. `<h3>` should be used here.

Reply:
Fixed